chameleon_mailman/models.py

```diff
@@ -6,10 +6,17 @@
 from django.utils.translation import ugettext, ugettext_lazy as _
 from django.utils.html import strip_tags
 
+from .tasks import schedule_email
+
+from datetime import timedelta
 import logging
 
 logger = logging.getLogger(__name__)
 
+
+SECONDS_BEFORE_REMINDER_EMAIL = (60 * 60 * 2)
```

Comment:
Should this read from the `OUTAGE_EMAIL_REMINDER_TIMEDELTA` defined in settings?

Reply:
Moved this to settings in clean up and forgot to remove old code.